### PR TITLE
Added checkbox for terms and conditions

### LINKED ISSUE #714: Users must accept Terms and Conditions before creating a group
- [ ] [text yet to be provided](https://www.notion.so/distributeaid/Feedback-session-18-02-22-ad2c1bad6f8c44c1b6f766c2fff86aaa?p=5517c09e4ac74d1ca4178c6fa4b768cc)
- [ ] frontend: add UI element to review and accept Terms and Conditions and notify backend
- [ ] backend: store time of acceptance in backend
- [ ] backend: group creation must not be possible if Terms and Conditions have not been accepted

### PR DESCRIPTION
**Pull request by pavle995**
Added checkbox for terms and conditions 
Prompting window with terms and conditions text
Sending time when it's checked to backend and storing it in postgres

(WIP) Missing terms and conditions text!

See #714

### CODE REVIEW

File: db/migrations/003-t&c.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { DataTypes, QueryInterface } from 'sequelize'
+
+export const up = async (queryInterface: QueryInterface) => {
+  await queryInterface.addColumn(`Groups`, 'termsAndConditionsAcceptedAt', {
+    allowNull: false,
+    type: DataTypes.DATE,
+  })
```
Comment (coderbyheart) [inline db/migrations/003-t&c.ts:7]: **problem:** this needs to be a **User** property, not a **Group** property.

File: frontend/src/pages/groups/TermAndCondCheckbox.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        label=""
+        checked={timeTcChecked === null ? false : true}
+        onChange={handleTcChange}
+        className="cursor-pointer"
+      />
+      <a
+        className="cursor-pointer text-blue-500"
+        onClick={() => setShowTermsAndCond(true)}
+      >
+        &nbsp;I accept terms and conditions
```
Comment (coderbyheart) [inline frontend/src/pages/groups/TermAndCondCheckbox.tsx:27]: **nitpick**: use CSS to add spacing, not content.

**Review by coderbyheart — CHANGES_REQUESTED** [review]

File: db/migrations/003-t&c.ts (hunk 1 above)
Comment (pavle995) [reply db/migrations/003-t&c.ts:7]: Hmm, what if one user creates multiple groups?

Comment (coderbyheart) [reply db/migrations/003-t&c.ts:7]: That's fine. In that case they have already accepted the Terms & Conditions. They apply to humans, not to groups. Groups are legally represented by humans, so the humans have to accept the terms.

Comment (pavle995) [reply db/migrations/003-t&c.ts:7]: Thanks for explanation! I'll work on change and create new PR

**Comment by coderbyheart** [comment]
Closing in favour of #844

**Pull request closed without merging**